lib/JMAP/Tester.pm

```diff
@@ -366,6 +366,8 @@ sub request {
 
 sub munge_method_triple {}
 
+sub response_class { 'JMAP::Tester::Response' }
```

Comment:
ultra nit: Classes like this that return a constant in a method can be sped up (20 or more% or so) with an empty prototype (as they do become const-ified subs)